envs/example/defaults.yml

```diff
@@ -107,8 +107,11 @@ nova:
     debug: True
     verbose: True
 
+ceph:
+  enabled: false
+
 cinder:
-  enabled: True
+  enabled: {{ ceph.enabled }}
   volume_type: file
```

Comment:
Hmm yes, that might be tricky. New customers have to get ceph in order to receive cinder?